### PR TITLE
Face get center API

### PR DESCRIPTION
**Pull request by mlfarrell**
this diff is going to look screwed up due to me having to reverse jump back to get around pre-emptive release of 0.2.24.

### CODE REVIEW

File: modeling-cmds/src/def_enum.rs [hunk 1]
```diff
@@ -542,6 +542,13 @@ define_modeling_cmd_enum! {
             pub uv: Point2d<f64>,
         }
 
+        ///Obtains the surface "center of mass"
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariant)]
+        pub struct FaceGetCenter {
+            /// Which face is being queried.
+            pub object_id: Uuid,
```
Comment (adamchalmers) [inline modeling-cmds/src/def_enum.rs:549]: Shouldn't this be "face_id" as you're not querying an overall object, you're querying one specific face of an object?

File: modeling-cmds/src/def_enum.rs [hunk 2]
```diff
@@ -951,6 +958,14 @@ define_modeling_cmd_enum! {
             pub padding: f32,
         }
 
+        /// Fit the view to the scene with an isometric view.
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariantEmpty)]
+        pub struct ViewIsometric {
+            /// How much to pad the view frame by.
```
Comment (adamchalmers) [inline modeling-cmds/src/def_enum.rs:964]: What unit is the padding?

File: modeling-cmds/src/def_enum.rs [hunk 3]
```diff
@@ -951,6 +958,14 @@ define_modeling_cmd_enum! {
             pub padding: f32,
         }
 
+        /// Fit the view to the scene with an isometric view.
+        #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, ExecutionPlanFromMemory, ModelingCmdVariantEmpty)]
+        pub struct ViewIsometric {
```
Comment (adamchalmers) [inline modeling-cmds/src/def_enum.rs:963]: So this zooms out to fill all objects within the scene?

File: modeling-cmds/src/def_enum.rs (hunk 2 above)
Comment (mlfarrell) [reply modeling-cmds/src/def_enum.rs:964]: this already merged and is in main,  see my comment about the diff being confusing due to me having to base this off of earlier release + cherrypicks.   my diff is only for the face center stuff.

File: modeling-cmds/src/def_enum.rs (hunk 1 above)
Comment (mlfarrell) [reply modeling-cmds/src/def_enum.rs:549]: opted for consistency, see FaceGetGradient + FaceGetPosition which already use object_id

File: modeling-cmds/src/def_enum.rs (hunk 3 above)
Comment (mlfarrell) [reply modeling-cmds/src/def_enum.rs:963]: this was Serena's PR that already has merged into main.  again, read my top comment about the diff.

**Review by adamchalmers — APPROVED** [review]

**Comment by mlfarrell** [comment]
k, main merged, diff cleaned up so its more clear.

**Pull request merged**